modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/ContentDashboardManagementToolbarPropsTransformer.js

```diff
@@ -106,11 +97,22 @@ export default function propsTransformer({portletNamespace, ...otherProps}) {
 		openSelectionModal({
```

Comment:
Ya I thought about removing it yesterday, but wanted to check if we also need to remove it from the rest of the code base?  I see it listed at https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java#L196 and https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/selector/ContentDashboardItemTypeItemSelectorView.java#L109.

In the first scenario, it is passed to `ItemSelector.java` at https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/item-selector/item-selector-api/src/main/java/com/liferay/item/selector/ItemSelector.java#L107 which I think can be removed safely.

For the second scenario, it is passed into `ContentDashboardItemTypeItemSelectorViewDisplayContext.java` at https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardItemTypeItemSelectorViewDisplayContext.java#L26.

Do we remove all references to `itemSelectedEventName`?

Reply:
@jonmak08 Yeah, we should clean it up! I thing these places should be enough:
- Constructor / getter / private property from [ContentDashboardItemTypeItemSelectorViewDisplayContext](https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardItemTypeItemSelectorViewDisplayContext.java) . The only place where that getter is used from is deleted in [this PR](https://github.com/liferay-frontend/liferay-portal/pull/967/commits/411ff5b35c08776e371579b641a09ac4e08d2c26#diff-6831480cb21f17ea7a6411cd795df9c0c941b8d2335c43c45a52bde6bb055fc4L83).
- Initalization of that class, in [ContentDashboardItemTypeItemSelectorView](https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/selector/ContentDashboardItemTypeItemSelectorView.java#L105-L110), should only have the `searchContainer` argument.

Reply:
> I don't see how this is related to `itemSelectedEventName` 🤔

When I was looking at it, we have a custom select event listed inside of https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/ContentDashboardManagementToolbarPropsTransformer.js.  See line [L41](https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/ContentDashboardManagementToolbarPropsTransformer.js#L41), [L99](https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/ContentDashboardManagementToolbarPropsTransformer.js#L99), [L125](https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/ContentDashboardManagementToolbarPropsTransformer.js#L125), [L142](https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/ContentDashboardManagementToolbarPropsTransformer.js#L142).  Those events are called from the java file like https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java#L196, which is why I was asking what we wanted to do with that event being passed into `_itemSelector.getItemSelectorURL()`?  Because the method eventually calls https://github.com/jonmak08/liferay-portal/blob/master/modules/apps/item-selector/item-selector-api/src/main/java/com/liferay/item/selector/ItemSelector.java#L107.